.run_tests.sh

```diff
@@ -26,6 +26,12 @@ cd tests
 ./make_watertight_no_curve_sphere_tests
 ./make_watertight_cylinder_tests
 ./make_watertight_cone_tests
+# if this is not a pull request, run regression tests
+if [ ! TRAVIS_PULL_REQUEST ] ; then
+wget https://uwmadison.box.com/shared/static/jns2dfu47euucwexe74l4qroxneelp9u.gz -O mw_reg_test_files.tar.gz -o wget.out
```

Comment:
I thought we were going to obfusciate this link and keep in a Travis encrypted variable?